core/admin/assets/admin-menu-styles.css

```diff
@@ -496,10 +496,13 @@ div.TB-ee-frame a {
 
 .ee-icon-size-20,
 .ee-icon-size-20:before {
```

Comment:
@tn3rb I see that these class name is used in few files, I need to know if the change/fix I've introduced won't conflict or break smth:

![image](https://user-images.githubusercontent.com/1477453/89266986-4eb52700-d63f-11ea-94b4-0937579330a1.png)